src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/RazorPagesWebSite.SimpleForms.html

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Not sure what happened here. Doesn't seem to be related to my Razor changes. @dougbu is this a result of removing the `Trim()` above?

Reply:
No, this is the result of re-running with `GENERATE_BASELINES`. This file now contains the correct baseline text.

Reply:
I ask because this wasn't a test failure when I ran locally